static/js/components/ProfileImageUploader.js

```diff
@@ -24,6 +25,10 @@ const dropZone = (startPhotoEdit, setPhotoError) => (
   </Dropzone>
 );
 
+const uploaderBodyHeight = (): number => (
+  window.innerHeight / 2
```

Comment:
could you import `ramda` in this file and make the return value here `R.min(500, window.innerHeight / 2`? I think that should fix the dialog height issue.